lib/tasks/prettier.rake

```diff
@@ -1,18 +1,16 @@
-if Rails.env.development? || Rails.env.test?
-  file_glob = "'./**/*.{css,html,js,js.erb,json,md,scss}'"
+file_glob = "'./**/*.{css,html,js,js.erb,json,md,scss}'"
```

Comment:
it would be good to rework these tasks so that we have two sat in a `prettier` namespace and then this `file_glob` variable could be scoped to the namespace, rather than defined here globally, but not a blocker for this PR.